deployment.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: binary-bot
+  labels:
+    app: binary-bot
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: binary-bot
+  template:
+    metadata:
+      labels:
+        app: binary-bot
+    spec:
+      containers:
+        - name: binary-bot
```

Comment:
Need:
* livenessProbe
* readinessProbe
* resource limits and claims
* imagePullPolicy

Reply:
The work is still in progress :) Once it is done I will move this config to another repo.